controls/stig_rhel9.yml

```diff
@@ -469,17 +469,17 @@ controls:
         levels:
             - medium
         title: RHEL 9 must not have the ypserv package installed.
-        rules:
+        related_rules:
             - package_ypserv_removed
-        status: automated
+        status: not applicable # The ypserv package is not available in RHEL 9
```

Comment:
In other PRs, I left this as automated and just moved it related rules. 

We should decide if we like this approach better. 

I think I like this better.